### PR TITLE
In *CTR and *STR documentation, update thresholds as the default ones

### LINKED ISSUE #351: In *CTR and *STR documentation, update thresholds
Dear @lindadelacombaz , 

you spotted that thresholds in the documentation for the indicators do not align with the actual default thresholds in the indicator code. Please update the documentation accordingly. This has low priority, as we did not share the code and documentation with banks yet.

### PR DESCRIPTION
**Pull request by lindadelacombaz**
Closes #351.

Dear @Tilmon and @kalashsinghal (cc @maurolepore),

I have updated the thresholds in the .Rmd files as the default ones in the code (30% to 30.33% and 70% to 66.66%, as discussed with you @Tilmon.

@kalashsinghal, if you want you can review this PR too, I have assigned you as a reviewer so you are updated with my tickets ;) 

Thanks ! 

----

TODO

- [x] [Link related issue/PR]([url](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)). 
- [x] Describe the goal of the PR. Avoid details that are clear in the diff.
- [x] Mark the PR as draft.
- [x] Review your own PR in "Files changed".
- [x] Ensure the PR branch is updated.
- [x] Ensure the checks pass.
- [x] Change the status from draft to ready.
- [x] Polish the PR title and description.
- [x] Assign a reviewer.

EXCEPTIONS

- [x] Slide here any item that you intentionally choose to not do.
- [ ] [Include a unit test](https://code-review.tidyverse.org/reviewer/aspects.html#sec-tests).

### CODE REVIEW

File: inst/extdata/child/intro-istr.Rmd [hunk 1]
```diff
@@ -8,11 +8,11 @@ needs to produce its products.The input products are, for example, resources,
...
 to produce the product.
 
-If an input product belongs to a sector in which reduction targets are over 70%,
+If an input product belongs to a sector in which reduction targets are over 66.66%, 
 it belongs to a high-risk sector and is classified as 'high sector transition
 risk'. If the input product belongs to a sector in which reduction targets are
-between 30-70%, the product is classified into the category 'medium sector
-transition risk' and if it is below 30% it belongs to the category of 'low
+≥33.33% and <66.66%, the product is classified into the category 'medium sector
+transition risk' and if it is strictly below 33.33% it belongs to the category of 'low
```
Comment (Tilmon) [inline inst/extdata/child/intro-istr.Rmd:15]: If above mentioned comment holds true, delete "strictly" as it's <= 33.33%

File: man/istr.Rd [hunk 2]
```diff
@@ -47,11 +47,11 @@ needs to produce its products.The input products are, for example, resources,
 packaging materials, energy and enabling services (such as tractor use on farm)
 to produce the product.
 
-If an input product belongs to a sector in which reduction targets are over 70\%,
-it belongs to a high-risk sector and is classified as 'high sector transition
+If an input product belongs to a sector in which reduction targets are strictly
+over 66.66\%, it belongs to a high-risk sector and is classified as 'high sector transition
```
Comment (Tilmon) [inline man/istr.Rd:51]: @lindadelacombaz I don't know why there was a "\" in the first place... Do you know? If not, please get rid of the "\" in front of %-signs in the whole article. 
Also, after double checking how <= / < are set, please eventually update the signs as above.

File: inst/extdata/child/thresholds.Rmd [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Products in the highest percentile (greater than `high_threshold`) are classified 
+as high transition risk products. Products in the medium percentile (greater 
+than `low_threshold` and lower or equal than `high_threshold`) are classified as 
+medium transition risk products. Products in the lowest percentile (lower or 
+equal than `low_threshold`) are classified as low transition risk products. 
+For details about the default low/high_thresholds, refer to the documentation of
```
Comment (maurolepore) [inline inst/extdata/child/thresholds.Rmd:6]:
```suggestion
For details about the default `low/high_thresholds`, refer to the documentation of 
```

File: inst/extdata/child/thresholds.Rmd [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Products in the highest percentile (greater than `high_threshold`) are classified 
+as high transition risk products. Products in the medium percentile (greater 
+than `low_threshold` and lower or equal than `high_threshold`) are classified as 
+medium transition risk products. Products in the lowest percentile (lower or 
+equal than `low_threshold`) are classified as low transition risk products. 
+For details about the default low/high_thresholds, refer to the documentation of 
+the corresponding **tr() function ([example](https://2degreesinvesting.github.io/tiltIndicator/reference/xctr.html#ref-usage)).
```
Comment (maurolepore) [inline inst/extdata/child/thresholds.Rmd:7]:
```suggestion
the corresponding `**tr()` function ([example](https://2degreesinvesting.github.io/tiltIndicator/reference/xctr.html#ref-usage)).
```

**Review by maurolepore — APPROVED** [review]
LGTM. 

Thanks @lindadelacombaz!

I did tiny suggestions about formatting. I plan to merge them myself. I'll check the links but after merging. Thanks for driving my attention there.

**Pull request merged**